### ISSUE TITLE
1.18.2 non-functional

### ISSUE THREAD

**Issue by Dantasstic** [ElPumpo/TinyNvidiaUpdateChecker#222]
The app simply flashes open and closed instantly upon launch.

I was previously using 1.17.1, which still works properly.

**Comment by Technetium1** [comment]
@Dantasstic if you run it from a terminal, is there lingering output?

**Comment by Connexiumlabs** [comment]
Same issue here with all versions starting with [1.18.0]
Double click right from Windows explorer in my download folder and beside the DO NOT RUN popup that I skip, it just flashes. 
1.17.1 runs fine.

Thank you

**Comment by ElPumpo** [comment]
@Connexiumlabs again, run TNUC with a terminal such as CMD to see the message/error

**Comment by Connexiumlabs** [comment]
Dah! 🙄 apologize; the error is due to missing/outdated .NET framework.

> 
> You must install or update .NET to run this application.
> 
> App: C:\Users\Patrick\Downloads\TinyNvidiaUpdateChecker.exe
> Architecture: x86
> Framework: 'Microsoft.NETCore.App', version '8.0.0' (x86)
> .NET location: C:\Program Files (x86)\dotnet\
> 
> The following frameworks were found:
>   7.0.20 at [C:\Program Files (x86)\dotnet\shared\Microsoft.NETCore.App]
> 
> Learn about framework resolution:
> https://aka.ms/dotnet/app-launch-failed
> 
> To install missing framework, download:
> https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=8.0.0&arch=x86&rid=win10-x86
> 

Once installed, program worked.
But I had to install the x86 version. x64 would not work (if someone reads this).

Thank you for the support and your work.
Maybe a check with a error message would help future people like me?

Cheers

**Comment by ElPumpo** [comment]
Yea I'm sorry guys I was told .NET 8 added a new prompt for this fucking thing, but they haven't! I could still downgrade to .NET 7 until 8 gets more mainstream... not sure.

**Comment by Dantasstic** [comment]
> run TNUC with a terminal such as CMD to see the message/error

FYI, running it with the start command from either CMD or PowerShell provides no extra info on why it doesn't run.

**Comment by Dantasstic** [comment]
@ElPumpo Long time users don't see the main GitHub page, and need to be informed via the download section for the new update on GitHub when new requirements are needed for the app, otherwise, well, this happens XD

As the app update popup simply asks if we want to go to the download page, we need to see the new requirements stated where it takes us.

![2024-06-10 - 1949-26](https://github.com/ElPumpo/TinyNvidiaUpdateChecker/assets/48727187/790c705a-ae1c-4e55-94be-af9da88592fe)

![2024-06-10 - 1949-39](https://github.com/ElPumpo/TinyNvidiaUpdateChecker/assets/48727187/0ba6f3e3-6414-4554-a2ef-8f4d66ca5525)

**Comment by ElPumpo** [comment]
Yea there is no way to fix that. Again I assumed they added a popup which they didn't.

**Comment by Dantasstic** [comment]
> Yea there is no way to fix that.

...what? You write what is on the download release page. Just write the requirements when they change....

I didn't say to change the popup I said to write the new requirements on your release page.

**Comment by ElPumpo** [comment]
Yea I forgot to mention 1.18.3 will come with a revisited update dialog. It will display a changelog. It should have been that way since day 1 I know. Ask Hawaii from 2017 why its like this

**Comment by Connexiumlabs** [comment]
You guys are the best. I appreciate all the work you do. People tend to forget you are doing this in your free time and dont actually us any of us anything hehe.

Wishing you and this software the best!

**Comment by dyni00** [comment]
not work for me.. update net7 + net8

**Comment by ElPumpo** [comment]
@dyni00 run TNUC using a terminal to show error message

**Comment by dyni00** [comment]
> @dyni00 run TNUC using a terminal to show error message

Weird. it didn't work yesterday. i download 1.18.2 and running, and 1.18.1 It stopped working! 

reinstalled runtime libraries net7 and 8 (but x64) and after running TNUC 1.18.3 in the terminal it showed that I did not have runtime lib 8, I downloaded x86 according to the link from terminal and it works! .... 

thx for app. the best app in the world!!!!!!!!
